templates/auth/dashboard/user_detail.html

```diff
@@ -97,11 +101,11 @@
             <div class="panel-body">
                 <div class="col-md-6">
                     <h4>Allergier</h4>
-                    <p>{{ user.allergies }}</p>
```

Comment:
Use `{{ user.allergies|default:"-" }}` instead?
Might just be odd since most of the other fields need if blocks.